src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+const convertToObject = (stylesString) => {
+  const styles = {};
+
+  const declarations = stylesString.split(';');
+
+  declarations.forEach((declaration) => {
+    const trimmedDeclaration = declaration.trim();
+
+    if (trimmedDeclaration) {
+      const [property, value] = trimmedDeclaration.split(':');
+
+      const trimmedProperty = property.trim();
+      const trimmedValue = value.trim();
+
+      styles[trimmedProperty] = trimmedValue;
+    }
+  });
```

Comment:
Instead of using `forEach` to iterate and build the `styles` object, you could use `reduce` which is more suited for this kind of transformation from an array to an object. This would also align with the checklist's recommendation to avoid modifying an existing object within iteration methods.